netlify.toml

```diff
@@ -2,3 +2,17 @@
   base = "site/"
   command = "jekyll build"
   publish = "site/_site"
+[build.processing]
+  skip_processing = false # -> Enable processing
+[build.processing.css]
+  bundle = true # -> Concatenate consecutive CSS files together to reduce HTTP requests.
```

Comment:
Jekyll already runs the css through sass and produces a single style css. Is this needed?

Reply:
After reviewing the live site and test site from Netlify I can see that the CSS bundling and minify process from Netlify doesn't appear to have any impact.

I can remove those from the toml file if you would like